Pinta.Tools/Tools/EraserTool.cs

```diff
@@ -118,26 +118,20 @@ protected override void OnSaveSettings (ISettingsService settings)
 			settings.PutSetting (ERASER_TYPE_SETTING, type_combobox.ComboBox.Active);
 	}
 
-	[MemberNotNull (nameof (lut_factor))]
-	private void InitLookupTable ()
+	private static IReadOnlyList<IReadOnlyList<byte>> CreateLookupTable ()
 	{
-		if (lut_factor is not null)
-			return;
-
-		lut_factor = new byte[LUT_Resolution + 1][];
-
+		var result = new byte[LUT_Resolution + 1][];
 		for (var dy = 0; dy < LUT_Resolution + 1; dy++) {
-			lut_factor[dy] = new byte[LUT_Resolution + 1];
```

Comment:
Looking more closely at the original code, I'm not sure why it used a jagged array (`byte[][]`) when all of the rows are the same size. Using a normal multidimensional array (`byte[,]`) would be better IMO since it avoids a bunch of separate smaller allocations

Reply:
@cameronwhite I just replaced it with a 2D array